puppet/modules/web/manifests/vhost/yum.pp

```diff
@@ -63,14 +63,20 @@
     }
   }
 
-  ['HEADER.html', 'robots.txt', 'RPM-GPG-KEY-foreman'].each |$filename| {
```

Comment:
I'd manually remove the file from web01, writing an absent section for one box seemed odd